sql/item_mods.sql

```diff
@@ -36192,6 +36192,12 @@ INSERT INTO `item_mods` VALUES (26341,9,20);      -- DEX+20
 INSERT INTO `item_mods` VALUES (26341,160,-600);    -- Damage Taken-6
 INSERT INTO `item_mods` VALUES (26341,302,8);     -- Triple Attack+8
 INSERT INTO `item_mods` VALUES (26341,973,15);    -- Subtle Blow II +15
+INSERT INTO `item_mods` VALUES (26356,30,5);    -- Magic Accuracy + 5
```

Comment:
missing the item name, see the example of `Culminus: DEF+60` below.

Reply:
ok there was a reason I directed you to the example below..all the other items only have their name on the first row on their mods. not all of the mod rows.

Reply:
I see it now, I thought you changed to more data and those where older item not yet changed